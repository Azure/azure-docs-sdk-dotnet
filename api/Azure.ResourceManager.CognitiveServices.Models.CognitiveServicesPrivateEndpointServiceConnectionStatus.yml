### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus&text=CognitiveServicesPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.CognitiveServices.Models
name: CognitiveServicesPrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.CognitiveServices.dll
packagesWithMoniker:
- value: Azure.ResourceManager.CognitiveServices v1.3.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.CognitiveServices v1.3.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct CognitiveServicesPrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type CognitiveServicesPrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure CognitiveServicesPrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of CognitiveServicesPrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.CognitiveServices.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::CognitiveServices::Models::CognitiveServicesPrivateEndpointServiceConnectionStatus
  - CognitiveServicesPrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
