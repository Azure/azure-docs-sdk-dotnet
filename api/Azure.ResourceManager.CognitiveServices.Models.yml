### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.CognitiveServices.Models.AbusePenalty
  monikers:
  - azure-dotnet
- uid: Azure.ResourceManager.CognitiveServices.Models.ArmCognitiveServicesModelFactory
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.AvailableCognitiveServicesSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountDeploymentModel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountDeploymentProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountDeploymentScaleSettings
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountModel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesCapacityConfig
  monikers:
  - azure-dotnet
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesCommitmentPlanPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesDomainAvailabilityContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesDomainAvailabilityList
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesIPRule
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesKeyVaultProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesModel
  monikers:
  - azure-dotnet
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesModelSku
  monikers:
  - azure-dotnet
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesMultiRegionSettings
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesNetworkRuleSet
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPatchResourceTags
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateLinkResource
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateLinkResourceProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesRegionSetting
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuAvailabilityContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuAvailabilityList
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuCapability
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuChangeInfo
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuRestrictionInfo
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuRestrictions
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesVirtualNetworkRule
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentCost
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentPeriod
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentPlanAssociation
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentPlanProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentQuota
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentTier
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.RegenerateServiceAccountKeyContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountApiKeys
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountApiProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountCallRateLimit
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountEncryptionProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountModelDeprecationInfo
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountQuotaLimit
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountThrottlingMatchPattern
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountThrottlingRule
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountUsage
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountUsageMetricName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountUserOwnedStorage
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
enums:
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuRestrictionsType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountKeyName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.CognitiveServices.Models.AbusePenaltyAction
  monikers:
  - azure-dotnet
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountDeploymentProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesAccountDeploymentScaleType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesNetworkRuleAction
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesRoutingMethod
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuRestrictionReasonCode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuTier
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.CommitmentPlanProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.DeploymentModelVersionUpgradeOption
  monikers:
  - azure-dotnet
- uid: Azure.ResourceManager.CognitiveServices.Models.ModelLifecycleStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountEncryptionKeySource
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountHostingModel
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountPublicNetworkAccess
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountQuotaUsageStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CognitiveServices.Models.ServiceAccountUsageUnitType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.CognitiveServices.Models
commentId: N:Azure.ResourceManager.CognitiveServices.Models
name: Azure.ResourceManager.CognitiveServices.Models
fullName: Azure.ResourceManager.CognitiveServices.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.CognitiveServices.Models
  - Azure::ResourceManager::CognitiveServices::Models
  - Models
  - CognitiveServices.Models
  - CognitiveServices::Models
  helpviewer_keywords:
  - Azure.ResourceManager.CognitiveServices.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
