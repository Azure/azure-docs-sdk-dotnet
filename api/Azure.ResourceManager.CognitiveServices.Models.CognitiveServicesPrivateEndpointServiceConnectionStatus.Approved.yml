### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus&text=CognitiveServicesPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  nameWithType: CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As CognitiveServicesPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.CognitiveServices.Models
name: Approved
fullName: Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
nameWithType: CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.CognitiveServices.dll
packagesWithMoniker:
- value: Azure.ResourceManager.CognitiveServices v1.3.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.CognitiveServices v1.3.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.CognitiveServices.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::CognitiveServices::Models::CognitiveServicesPrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved
  - CognitiveServicesPrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - CognitiveServicesPrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct CognitiveServicesPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
